src/components/Forms/Input/input.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useField } from 'formik';
+
+export const Input = ({
+  id, label, ...props
```

Comment:
i think it will be better to distructure `errors` and `touched` also

Reply:
fixed, thanks